Polygon_repair_2/test/Polygon_repair_2/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+cmake_minimum_required(VERSION 3.1...3.23)
+project(Polygon_repair_2_Tests)
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
this is not needed

Reply:
what should I do to compile with C++17 then?

Reply:
If you merge master, then it will be set ON by default